drivers/bluetooth/hci/Kconfig

```diff
@@ -71,6 +76,50 @@ config BT_NO_DRIVER
 
 endchoice
 
+if BT_H4_UART
+
+module = BT_H4_UART
+module-str = H4 UART transport
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+choice
+	prompt "UART API"
+
+config BT_H4_UART_INTERRUPT_DRIVEN
```

Comment:
Why do we need this config option? Have we ever supported h4 on a UART that only has poll_in and poll_out?

Reply:
Oh, I see, it's a choice, sorry.